go/src/meguca/common/errors.go

```diff
@@ -82,6 +82,11 @@ func ErrInvalidBoard(board string) error {
 	return StatusError{fmt.Errorf("board `%s` does not exist", board), 404}
 }
 
+// Missing string
+func ErrMissingString(str string) error {
```

Comment:
No need for an extra public function, if you only use it once. Just inline the StatusError creation in getHookTubeTitle().